### PR TITLE
[EDU-4499] feat: add WAAP architecture

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

- Added diagram image
- Added architecture text

### Additional links

Compare with original version: https://docs.google.com/document/d/1OMIQff-lhtV9ULJzApHBLp0qBCJUeyA2wGyHws18O0E/edit

[EDU-4499]: https://aziontech.atlassian.net/browse/EDU-4499?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/architectures/waap-architecture.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+title: Enhance Cybersecurity with Azion's Advanced WAAP Solution
+description: Use Azion WAAP solution to secure applications and APIs against cyber threats from OWASP Top 10, sophisticated zero-day attacks, and automated bot activities.
+meta_tags: architecture, secure, firewall, waap, web application and api protection, waf
+namespace: docs_arch_waap
+permalink: /documentation/architecture/edge-firewall/web-application-and-api-protection-waap/
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/architectures/waap-architecture.mdx:6]: dalee

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/en/pages/architectures/waap-architecture.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+---
+
+[Azion's Web Application and API Protection (WAAP)]() ensures robust defense mechanisms against cyber threats such as OWASP Top 10, sophisticated zero-day attacks, and automated bot activities. This platform ensures the security and uninterrupted availability of web applications and APIs by employing a comprehensive suite of integrated security features that meticulously analyze and manage incoming requests to protect against a wide range of attacks.
```
[the remark this answers is not in the document]
Comment (MarianaAguilera) [reply src/content/docs/en/pages/architectures/waap-architecture.mdx:9]: não deveria ser: this architecture ou this solution?

File: src/content/docs/en/pages/architectures/waap-architecture.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+This solution is ideal for organizations looking to safeguard their digital infrastructure against advanced security threats, providing reliable protection for web applications and APIs against the most challenging cyber attacks.
+
+## Web Application and API Protection (WAAP) Architecture Diagram
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/waap-architecture.mdx:13]:
```suggestion

---

## Web Application and API Protection (WAAP) Architecture Diagram
```

File: src/content/docs/en/pages/architectures/waap-architecture.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+![Architecture diagram](/assets/docs/images/uploads/diagram-waap.png)
+
+### Web Application and API Protection (WAAP) Dataflow
+
+1. A request reaches Azion Edge Platform and is directed to Edge Firewall.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/waap-architecture.mdx:19]:
```suggestion
1. A request reaches Azion Edge Platform and is redirected to Edge Firewall.
```

File: src/content/docs/en/pages/architectures/waap-architecture.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+2. Edge Firewall processes the request, where DDoS Protection Unmetered assesses for potential DDoS or DoS attacks, blocking or allowing the request accordingly.
+3. Edge Firewall's Rules Engine processes the rules during the request phase; if conditions are met, the corresponding Behaviors are executed.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/waap-architecture.mdx:21]:
```suggestion
3. Edge Firewall's Rules Engine processes the rules during the request phase; if conditions are met, the corresponding *behaviors* are executed.
```

File: src/content/docs/en/pages/architectures/waap-architecture.mdx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+### Components
+
+- [Edge Firewall](/en/documentation/products/secure/edge-firewall/): provides main settings to set up your instance and associate it with domains and activate modules.
+    - [DDoS Protection](en/documentation/products/secure/edge-firewall/ddos-protection/): detects and mitigates DDoS and DoS attacks when it's activated.
+    - [Web Application Firewall](/en/documentation/products/secure/edge-firewall/web-application-firewall/): allows you to configure protection against attacks and vulnerabilities found in OWASP Top 10 and block requests.
+    - [Network Layer Protection](/en/documentation/products/secure/edge-firewall/network-layer-protection/): allows you to create lists with IP addresses or countries with suspicious behavior and restrict their access.
+    - [Bot Manager](/en/documentation/products/guides/bot-manager/): identifies and classifies the request according to the intent-based behavioral analysis.
+- [Edge Application](/en/documentation/products/build/edge-application/): runs your application on top of the Azion Edge Platform.
+
+## Implementation
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/waap-architecture.mdx:40]:
```suggestion

---

## Implementation
```

File: src/content/docs/en/pages/architectures/waap-architecture.mdx [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+    - [Azion CLI](/en/documentation/products/azion-cli/overview/)
+- [Create an edge firewall](/en/documentation/products/guides/secure/edge-firewall-configure-main-settings/) and associate it with the configured domain, so all security rules apply to that specific domain.
+    - Enable the DDoS Protection Unmetered module.
+    - Enable the Web Application Firewall module.
+    - Enable the Network Layer Protection module.
+- [Create a WAF rule set](/en/documentation/products/guides/secure/create-waf-rule-set/).
+    - Configure Threat Type and Sensitivity levels.
+- [Create a network list in Network Layer Protection](/en/documentation/products/guides/blocklists-ip-addresses-edge/).
+    - Configure the IPs or countries that'll be blocked.
+- [Configure the Bot Protection integration](/en/documentation/products/guides/bot-protection/).
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/waap-architecture.mdx:54]:
```suggestion
- [Configure the Bot Manager integration](/en/documentation/products/guides/bot-manager/).
```

File: src/content/docs/en/pages/architectures/waap-architecture.mdx [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+    - Get the integration and set it up on Edge Firewall.
+- [Create rules in Rules Engine for Edge Firewall](/en/documentation/products/guides/secure/work-with-rules-engine/).
+    - Configure the Set WAF Rule Set behavior.
+    - Configure the Network criteria.
+    - Configure the [Request URI](/en/documentation/products/guides/bot-protection/#setting-up-the-rules-engine) criteria.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/waap-architecture.mdx:59]:
```suggestion
    - Configure the [Request URI](/en/documentation/products/guides/bot-manager/#setting-up-the-rules-engine) criteria.
```

File: src/content/docs/en/pages/architectures/waap-architecture.mdx [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+## Related docs
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/waap-architecture.mdx:61]:
```suggestion

---

## Related docs
```

File: src/content/docs/en/pages/architectures/waap-architecture.mdx [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+    - [Network Layer Protection](/en/documentation/products/secure/edge-firewall/network-layer-protection/): allows you to create lists with IP addresses or countries with suspicious behavior and restrict their access.
+    - [Bot Manager](/en/documentation/products/guides/bot-manager/): identifies and classifies the request according to the intent-based behavioral analysis.
+- [Edge Application](/en/documentation/products/build/edge-application/): runs your application on top of the Azion Edge Platform.
+
+## Implementation
+
+- Create an edge application using one of the recommended options:
+    - [Dynamic and Static File Optimization template](/en/documentation/products/guides/dynamic-and-static-file-optimization-template/)
+    - [Azion API](https://api.azion.com/#d43c5509-fb99-4b20-a7bd-64f0924af352)
+    - [Azion CLI](/en/documentation/products/azion-cli/overview/)
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/waap-architecture.mdx:45]:
```suggestion
    - [Dynamic and Static File Optimization template](/en/documentation/products/guides/dynamic-and-static-file-optimization-template/).
    - [Azion API](https://api.azion.com/#d43c5509-fb99-4b20-a7bd-64f0924af352).
    - [Azion CLI](/en/documentation/products/azion-cli/overview/).
```

File: src/content/docs/en/pages/architectures/waap-architecture.mdx [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+    - Bot Manager classifies the device, user, or intent as malicious.
+    - Security Edge Functions or custom rules, such as rate limiting, deny access or take other preventive actions to protect the application.
+5. Rules Engine permits the request to proceed to Edge Application.
+6. Edge Application's Rules Engine forwards the request to the Origin—whether it be multi-cloud, on-premises, or hybrid—and, upon receiving the response, initiates the Rules Engine's response phases.
+7. The client receives the final response.
+
+### Components
+
+- [Edge Firewall](/en/documentation/products/secure/edge-firewall/): provides main settings to set up your instance and associate it with domains and activate modules.
+    - [DDoS Protection](en/documentation/products/secure/edge-firewall/ddos-protection/): detects and mitigates DDoS and DoS attacks when it's activated.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/waap-architecture.mdx:34]:
```suggestion
    - [DDoS Protection](/en/documentation/products/secure/edge-firewall/ddos-protection/): detects and mitigates DDoS and DoS attacks when it's activated.
```

File: src/content/docs/en/pages/architectures/waap-architecture.mdx [hunk 12]
```diff
@@ -0,0 +1,69 @@
+---
+title: Enhance Cybersecurity with Azion's Advanced WAAP Solution
+description: Use Azion WAAP solution to secure applications and APIs against cyber threats from OWASP Top 10, sophisticated zero-day attacks, and automated bot activities.
+meta_tags: architecture, secure, firewall, waap, web application and api protection, waf
+namespace: docs_arch_waap
+permalink: /documentation/architecture/edge-firewall/web-application-and-api-protection-waap/
+---
+
+[Azion's Web Application and API Protection (WAAP)]() ensures robust defense mechanisms against cyber threats such as OWASP Top 10, sophisticated zero-day attacks, and automated bot activities. This architecture guarantees the security and uninterrupted availability of web applications and APIs by employing a comprehensive suite of integrated security features that meticulously analyze and manage incoming requests to protect against a wide range of attacks.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/waap-architecture.mdx:9]:
```suggestion
[Azion's Web Application and API Protection (WAAP)](https://www.azion.com/en/solutions/waap/) ensures robust defense mechanisms against cyber threats such as OWASP Top 10, sophisticated zero-day attacks, and automated bot activities. This architecture guarantees the security and uninterrupted availability of web applications and APIs by employing a comprehensive suite of integrated security features that meticulously analyze and manage incoming requests to protect against a wide range of attacks.
```

File: src/content/docs/pt-br/pages/arquiteturas/waap-arquitetura.mdx [hunk 13]
```diff
@@ -0,0 +1,67 @@
+---
+title: Reforce a segurança cibernética com a solução avançada de WAAP da Azion
+description: Utilize a solução WAAP da Azion para proteger aplicações e APIs contra ameaças cibernéticas do OWASP Top 10, ataques de dia zero e bots.
+meta_tags: arquitetura, secure, firewall, waap, proteção de aplicações e APIs, waf
+namespace: docs_arch_waap
+permalink: /documentacao/arquitetura/edge-firewall/web-application-and-api-protection-waap/
+---
+
+A [proteção de aplicações web e APIs]() (*Web Application and API Protection, WAAP*) da Azion proporciona mecanismos de defesa robustos contra ameaças cibernéticas como o OWASP Top 10, ataques sofisticados de dia zero e atividades automatizadas de bots. Esta plataforma garante a segurança e disponibilidade ininterrupta de aplicações web e APIs ao empregar uma ampla gama de recursos de segurança integrados que analisam e gerenciam meticulosamente as requisições recebidas para proteger contra diversos tipos de ataques.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/waap-arquitetura.mdx:9]:
```suggestion
A [proteção de aplicações web e APIs](https://www.azion.com/pt-br/solucoes/waap/) (*Web Application and API Protection, WAAP*) da Azion proporciona mecanismos de defesa robustos contra ameaças cibernéticas como o OWASP Top 10, ataques sofisticados de dia zero e atividades automatizadas de bots. Esta plataforma garante a segurança e disponibilidade ininterrupta de aplicações web e APIs ao empregar uma ampla gama de recursos de segurança integrados que analisam e gerenciam meticulosamente as requisições recebidas para proteger contra diversos tipos de ataques.
```

**Pull request merged**